### PR TITLE
Add dummy implementation for client command `match_ready` (#607)

### PR DESCRIPTION
**Pull request by Katharsas**
Moves #607 forward (see https://github.com/FAForever/downlords-faf-client/issues/1783#issuecomment-894844963).

Sadly untested because if currently don't have a local environment for running the python server, so i would be glad if somebody else could test that. Sorry.

### CODE REVIEW

**Comment by Askaholic** [comment]
Can you also add a unit test to `unit_tests/test_lobbyconnection.py`? It doesn’t need to do much just trigger the command to make sure it doesn’t raise any exceptions (see other examples in that file)

**Comment by Katharsas** [comment]
Of couse, this shall present no problem for me!

File: tests/unit_tests/test_lobbyconnection.py [hunk 1]
```diff
@@ -957,6 +957,10 @@ async def test_command_game_matchmaking_not_party_owner(
 
     lobbyconnection.ladder_service.cancel_search.assert_called_once()
 
+async def test_command_match_ready(lobbyconnection):
```
Comment (Askaholic) [inline tests/unit_tests/test_lobbyconnection.py:960]: Please put 2 blank lines between functions, otherwise my linter will yell at me.

Comment (Askaholic) [reply tests/unit_tests/test_lobbyconnection.py:960]: *Top level functions only. If they're methods of a class then only 1 line

**Pull request merged**